recipes/blend2d/all/conandata.yml

```diff
@@ -1,28 +1,33 @@
 sources:
+  "0.11.1":
+    url: "https://blend2d.com/download/blend2d-0.11.1.tar.gz"
+    sha256: "f46d61b6aa477fea1a353a41f5906d4e861817ae059ed22fc6ecdd50ff859dd2"
+    # When upstream will have new release, tarball may be deleted.
+    # Use following url: https://github.com/blend2d/blend2d/archive/a7f947677e85441abe1769586d3bb935b4086a07.tar.gz
```

Comment:
Thanks a _ton_ :)

I wonder why they feel the need to make these changes, seems super counterproductive to me (Also note that once we release https://blog.conan.io/2023/10/03/backup-sources-feature.html to the general userbase, things like these will be super mitigated, as we'll be able to provide the deleted source tarballs!)